addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -171,6 +173,7 @@ class ComposerTextInput extends Component {
      * @private
      */
     _updateHeight() {
+        this._mirroredTextareaRef.el.value = this.composer.textInputContent;
```

Comment:
This is not update height concern. I assume it's because you want to have it from `_onInputTextarea`? If so, I would prefer duplicate LOC or define new function/rename function.

Reply:
Writing on the mirror is just for height this is why I moved it.
The write on the real textarea is kept on _update.

Reply:
Ok, can ignore this comment then.